src/xcode/ENA/ENA/Source/Coordinator.swift

```diff
@@ -0,0 +1,280 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+import Connectivity
+
+protocol CoordinatorDelegate: AnyObject {
+	func coordinatorUserDidRequestReset()
+}
+
+class Coordinator: RequiresAppDependencies {
```

Comment:
It would be great to have some SwiftDoc comments here and also for the delegate protocol.

Reply:
Added documentation comments [here](https://github.com/corona-warn-app/cwa-app-ios/pull/834/commits/dbb1c8dd3d4d072e93a46c35edb0c25220356199).